src/theme/NativeCodeBlock/index.js

```diff
@@ -250,6 +251,9 @@ export default ({children, className: languageClassName, metastring}) => {
 
                   return (
                     <div key={i} {...lineProps}>
```

Comment:
nit: Keys should not be indices.

Reply:
@jjwill That code is from the native docusaurus component.